### DISCUSSION TITLE
Editor issue : href="tel:" getting sanitized

### DISCUSSION THREAD

**Discussion by smokinguns** [OrchardCMS/OrchardCore#7089] [Q&A] [answered]
I'm sure I'm missing something, but the href tag is being removed for tel: when in html body or .liquid.

for example i have an anchor tag in liquid template as follows.
<a href="tel:+14121231234">Call Us</a>

When i go back into the editor, the href tag is removed.

**Comment by hishamco** [comment, marked as answer]
Nothing wrong with `tel:` ;) this happen because the anchor tag is sanitized by default, you can allow disable the sanitization from the admin site

This quite similar to what have been discussed [here](https://github.com/OrchardCMS/OrchardCore/discussions/7034)

**Reply by smokinguns** [reply]
@hishamco thanks. I didn't remember seeing that option when i set it up. Makes perfect sens.

**Reply by smokinguns** [reply]
[previous discussion](https://github.com/OrchardCMS/OrchardCore/discussions/7034)

**Reply by hishamco** [reply]
I fixed the link, thanks ;)
